### ISSUE TITLE
Application will not start again after exiting from tray.

### ISSUE THREAD

**Issue by skewbug** [Taiko2k/Tauon#1237]
What the title says. I have the flatpak on Debian 12 and everything works fantastic except for this issue. Every time I open it for the first time after a reboot it functions as expected, however if I close the application fully it will not open again until I preform a restart.

**Comment by C0rn3j** [comment]
What happens when you launch it from a terminal?

Are you sure there isn't a hanging process? Check `htop` /  `ps aux`

**Comment by skewbug** [comment]
The process is no longer running after I quit the application as far as I can tell.
Trying to launch again from terminal returns 
`bwrap: Can't make symlink at /run/user/1000/discord-ipc-0: File exists`
so I'm guessing it has something to do with the discord status option. Currently not sure if the problem resolves itself upon disabling this option, but I'll edit with whatever I find as soon as I get a chance.

**Comment by C0rn3j** [comment]
Might have to blame Debian for this one, what does `bwrap --version` get you?

**Comment by skewbug** [comment]
Looks like it's 0.8.0
Is bubblewrap updatable or am I just going to have to wait for a new release?

**Comment by C0rn3j** [comment]
Everything is updatable, but you'll have to do it on your own or find a repo that already packaged it, since Debian is a fixed release distribution.

Debian 13 will have a working version, but that's probably about a year away.

`rm /run/user/1000/discord-ipc-0` might be a decent workaround until then.

Not much to do on Tauon's side.

**Comment by skewbug** [comment]
Removed discord-ipc-0 to make sure that was the issue and sure enough that fixed it! I went ahead and installed the latest release of bubblewrap, but is there some way I can make a new discord-ipc-0 symlink again to see if that restores all functionality? discord-ipc-0 seems to be created when installing stock discord, but is only valid while discord is open. This allows the stock discord app to work correctly with tauon again, but the symlink breaks when using any other discord client like vesktop. I apologize for my ignorance on this subject, but how can I return it to how it functioned previously?
